hawc/apps/assessment/serializers.py

```diff
@@ -38,6 +38,10 @@ def to_representation(self, obj):
         # obj is a model-manager in this case; convert to list to serialize
         return list(obj.values("slug", "name"))
 
+    class Meta:
```

Comment:
Meta class was required for OpenAPI, and since the serializing/deserializing methods are overwritten I don't believe this addition would break anything. I ran an API endpoint that used this serializer and I saw no difference.